Add --accept-only option to automatically accept single valid matches

apetresc opened the pull request:
As discussed in #180, if a search results in only a single valid match, even if the cover match doesn't meet the required threshold for an auto-match, if --accept-only is passed, then accept it anyway. This should result in a much smoother (less interactive) import for cases where the filename is already well-matched.

apetresc commented:
In addition to the unit test, manual testing also indicates this works as intended:

```bash
$ poetry run metron-tagger -co The\ X-Men\ \(1963\)\ \#142.cbz

Starting Online Search and Tagging:
-----------------------------------

The X-Men (1963) #142.cbz - Results found:
? Select an issue to match Skip
$ poetry run metron-tagger -co --accept-only The\ X-Men\ \(1963\)\ \#142.cbz

Starting Online Search and Tagging:
-----------------------------------
Writing 'ComicInfo.xml' metadata using 'The Uncanny X-Men #142 (1981)' for 'The X-Men (1963) #142.cbz'.

Successful Matches:
-------------------
The X-Men (1963) #142.cbz
```

(I know it _seems_ like this is a faulty match because it mapped `The X-Men #142` to `The Uncanny X-Men #142` but actually that's correct, it's just the weird back-and-forth naming of X-Men in those years! Those really are the same issue 😂)

bpepple reviewed (approved):
Looks good. The only change needed are fairly minor.

1. Make use of the talker fixture in the test
2. Run `make lint` and `make fix` to fix some minor linting errors.

I'll go ahead and make these when I merge the PR. Thx!

apetresc commented:
Addressed both those points – sorry about the test fixture thing, in an earlier revision of my code I was passing `accept_only` to the Talker's constructor so I couldn't use the fixture; but once I switched to keeping it in the args only, I forgot to undo that decision in the test.

Should be good now!
